src/main/java/io/github/sefiraat/networks/slimefun/network/NetworkQuantumStorage.java

```diff
@@ -232,7 +258,8 @@ public void init() {
                 for (int i : OUTPUT_SLOTS) {
                     addItem(i, BACK_OUTPUT, (p, slot, item, action) -> false);
                 }
-                addItem(ITEM_SET_SLOT, SET_ITEM, (p, slot, item, action) -> false);
+                ItemStack setItemItemstack = supportsCustomMaxAmount ? SET_ITEM_SUPPORTING_CUSTOM_MAX : SET_ITEM;
+                addItem(ITEM_SET_SLOT, setItemItemstack, (p, slot, item, action) -> false);
```

Comment:
Just a style choice (idk what sefi thinks is better), you could in-line that rather than declaring on another line

Reply:
Didn't wanna make the line really long and ugly

Reply:
Valid